website_sale_price_tier/__manifest__.py

```diff
@@ -5,7 +5,7 @@
 {
     'name': 'Website Sale - Price Tiers',
     'summary': 'Add price tier radio buttons to product pages in website shop',
-    'version': '10.0.1.0.0',
+    'version': '10.0.1.0.1',
```

Comment:
It should be 10.0.1.1.0, as it requires to update module.

Reply:
A module update is required, but the change is backwards compatible and is just a bugfix rather than a new feature, so changing the patch version seemed like the best option. That being said, I'm happy to use ``10.0.1.1.0`` instead.